ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -967,6 +1007,7 @@ module AuthADWinbind : Auth_signature.AUTH_MODULE = struct
       ; "--no-dns-updates"
       ]
       @ ou_param
+      @ dns_hostname_option
```

Comment:
`@` is not efficient and you need to be careful about precedence and associativity. Better use `List..concat`